src/util/backtrace.cc

```diff
@@ -95,11 +97,13 @@ frame decorate(uintptr_t addr) noexcept {
 
 simple_backtrace current_backtrace_tasklocal() noexcept {
     simple_backtrace::vector_type v;
+    #ifdef SEASTAR_HAVE_EXECINFO
```

Comment:
no need to `#ifdef` here, i think. as `backtrace()` would be a no-op if execinfo is not available.

Reply:
changed